src/main/java/picard/sam/RevertSam.java

```diff
@@ -155,8 +155,8 @@
     @Argument(shortName = "OBR", doc = "When true, outputs each read group in a separate file.")
     public boolean OUTPUT_BY_READGROUP = false;
 
-    @Argument(shortName = "RHC", doc = "Restore reads and qualities with hard-clips of records containing have XB and XQ tags.")
-    public boolean RESTORE_HARDCLIPS = false;
+    @Argument(shortName = "RHC", doc = "When false, does not attempt to restore reads and qualities of records with hard-clips containing XB and XQ tags.")
```

Comment:
Not sure why changing this option to default to `true` would require changing the description to be in the negative form. Usually it's easier to understand if documentation is written in the positive
```suggestion
    @Argument(shortName = "RHC", doc = "When true, attempt to restore reads and qualities of records with hard-clips containing XB and XQ tags.")
```